nautobot/extras/tests/test_views.py

```diff
@@ -1722,6 +1724,144 @@ def setUpTestData(cls):
             "task_queues": "overridden,priority",
             "task_queues_override": True,
         }
+        # This form is emulating the non-conventional JobBulkEditForm
+        cls.bulk_edit_data = {
+            "enabled": True,
+            "clear_grouping_override": True,
+            "grouping": "",
+            "clear_description_override": False,
+            "description": "Overriden Description",
+            "clear_dryrun_default_override": False,
+            "dryrun_default": "",
+            "clear_hidden_override": True,
+            "hidden": False,
+            "clear_approval_required_override": True,
+            "approval_required": True,
+            "clear_soft_time_limit_override": False,
+            "soft_time_limit": 350,
+            "clear_time_limit_override": True,
+            "time_limit": "",
+            "has_sensitive_variables": False,
+            "clear_has_sensitive_variables_override": False,
+            "task_queues": "overridden,priority",
+            "clear_task_queues_override": False,
+        }
+
+    def validate_job_data_after_bulk_edit(self, pk_list, old_data):
+        # Name is bulk-editable
+        overridable_fields = [field for field in JOB_OVERRIDABLE_FIELDS if field != "name"]
+        clear_override_fields = ["clear_" + field + "_override" for field in overridable_fields]
+        for instance in self._get_queryset().filter(pk__in=pk_list):
+            self.assertEqual(instance.enabled, True)
+            job_class = instance.job_class
+            if job_class is not None:
+                for clear_override_field in clear_override_fields:
+                    overridable_field = clear_override_field[6:-9]
+                    override_field = clear_override_field[6:]
```

Comment:
Comments please on where these numbers are coming from - I presume `6` is `clear_` and `-9` is `_override`?

So if I'm reading this right, for example:
- `field` is "grouping"
- `clear_override_field` is "clear_grouping_override"
- `overridable_field` is "grouping" (should we be iterating over `overridable_fields` instead of `clear_override_fields? Would that simplify any of this?)
- `override_field` is "grouping_override"